NuGet.Config

```diff
@@ -1,24 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <packageSources>
-        <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
-        <!-- Add repositories here to the list of available repositories -->
-        
-        <!-- Dependencies that we must carry because they're not on public nuget feeds right now. -->
-        <!--<add key="Static Package Dependencies" value="dep\packages" />-->

```

Comment:
Please leave a commented version of this in; i do use it from time to time when testing local MUX versions.

Reply:
OK.